consensus/propagation/reactor.go

```diff
@@ -200,14 +212,14 @@ func (blockProp *Reactor) RemovePeer(peer p2p.Peer, reason interface{}) {
 
 func (blockProp *Reactor) ReceiveEnvelope(e p2p.Envelope) {
 	if !blockProp.IsRunning() {
-		blockProp.Logger.Debug("Receive", "src", e.Src, "chId", e.ChannelID)
```

Comment:
unrelated, but didn't feel like this provided any useful information. will revert if reviewers have found this useful